dev-tools/ansible/roles/env_setup/tasks/main.yml

```diff
@@ -53,4 +53,8 @@
   service: name=firewalld state=started
   become: yes
 
+- name: open firewall port 22 for SSH connections
```

Comment:
Is this needed? If port 22 isn't open how will Ansible be able to connect to the host?

Reply:
Assume the machine do not have firewalld installed at the beginning and then we install firewalld on this. By default firewalld blocks all the ports. So it blocks next ssh connections if we do not whitelist port 22 in the same connection that installed firewalld.

Reply:
Yes, this makes sense. I was thinking of the OpenStack firewall. However, why have we never run into this issue previously?

Reply:
I tested in Virtualbox and I had to manually whitelist port 22 if it is not being done at this step.